english/travel-destinations/winter/winter.md

```diff
@@ -2,4 +2,5 @@ Winter can be a cold and bitter time. However, it is hard to find more
 beauty outdoors than the snow capped peaks of [Glacier National Park]
 (http://www.nps.gov/glac/index.htm)! Great adventure awaits there. You can
 camp and hike. If you are adventurous, you can try some winter wilderness
-treks in the [backcountry](http://www.nps.gov/glac/planyourvisit/backcountry.htm).
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.